src/v2/ion/ui-schema/ion-object-handler.js

```diff
@@ -18,7 +18,7 @@ import { AUTHENTICATOR_KEY } from '../RemediationConstants';
  * @param {AuthenticatorOption[]} options
  * @param {( AuthenticatorEnrollment[] || Authenticator[] )} authenticators
  */
-const createOVOptions = (options = []) => {
+function createOVOptions(options = []) {
```

Comment:
why this change ??

Reply:
if you scroll up the comments in the PR, Ani pointed out why https://github.com/okta/okta-signin-widget/pull/2006#discussion_r645057750